### PR TITLE
AT_025.005 | Header > Verify user will be redirected to new url "users/sign"

### PR DESCRIPTION
**Pull request by Vik1975**
Edit script according to comments.

### CODE REVIEW

**Comment by Vik1975** [comment]
https://trello.com/c/6jX8R7Ny/142-at025005-main-menu-dashboard-verify-the-first-button-try-the-dashbord-is-clickable-and-redirect-user-to-new-url-users-signin

**Review by alena-resh — APPROVED** [review]

File: cypress/e2e/jScript_group.cy.js [hunk 1]
```diff
@@ -139,6 +134,11 @@ describe('Group jScript_group', () => {
         cy.get('ul.day-list li').should('have.length', 8);
     });
 
+    it('AT_033.007 | Header > Navigation >> Verify "Pricing" menu link', () => {
```
Comment (tatsiana-yakimovich) [inline cypress/e2e/jScript_group.cy.js:137]: There are multiple problems here:
1. Don't use cy.visit('https://openweathermap.org') in your tests as we already doing it in beforeEach
2. AT_033.007 code is duplicated and not related to your merge. Probably, some merge conflict issue and you need to remove it.
3. Line 220 - you don't need those closing bracers

**Review by tatsiana-yakimovich — CHANGES_REQUESTED** [review]
Please address the comments that I just posted

File: cypress/e2e/jScript_group.cy.js (hunk 1 above)
Comment (TimothyHuxx) [reply cypress/e2e/jScript_group.cy.js:137]: @tatsiana-yakimovich Great comment. I would recommend requesting changes instead of writing a simple comment.

**Review by TimothyHuxx — CHANGES_REQUESTED** [review]
Why you deleted the test from lines 79-83? We don't delete tests. If we need to delete something we should create refactor card and do it separately.  
Also, there are two empty lines 78 and 79.
We add only one test in each PR

**Comment by TimothyHuxx** [comment]
@Vik1975 Got it. You still have a few issues. Please review the comment from tatsiana-yakimovich

**Comment by Vik1975** [comment]
@tatsiana-yakimovich Seems to have fixed everything. Could you check it please.

File: cypress/e2e/jScript_group.cy.js [hunk 2]
```diff
@@ -233,7 +225,7 @@ describe('Group jScript_group', () => {
         cy.get('#support-dropdown').click();
         cy.get('#support-dropdown-menu').should('be.visible');
         cy.get('#support-dropdown-menu a[href="/appid"]').click();
-
+        cy.url().should('eq','https://openweathermap.org/appid');
```
Comment (tatsiana-yakimovich) [inline cypress/e2e/jScript_group.cy.js:228]: Don't add such check as it's a duplicate and already there one line below.
Also, don't delete empty lines in other tests as they were added on purpose to separate different parts

**Review by tatsiana-yakimovich — CHANGES_REQUESTED** [review]
@Vik1975  Please take a look into new comments

File: cypress/e2e/jScript_group.cy.js (hunk 2 above)
Comment (Vik1975) [reply cypress/e2e/jScript_group.cy.js:228]: It is not my test you mentioned.

Comment (Vik1975) [reply cypress/e2e/jScript_group.cy.js:228]: I think it was merging issue. I did AT 025_004 and AT 025_005 only

Comment (Vik1975) [reply cypress/e2e/jScript_group.cy.js:228]: @tatsiana-yakimovich could you approve it please

Comment (tatsiana-yakimovich) [reply cypress/e2e/jScript_group.cy.js:228]: @Vik1975 we can't merge it in current state as you are changing someone's else tests which is not correct.
Not sure whether you are looking at the right place - just switch into "File changed" tab and review changes

**Pull request closed without merging**